packages/core/src/components/BreadcrumbsBar/__stories__/BreadcrumbsBar.mdx

```diff
@@ -101,10 +103,20 @@ import { BreadcrumbsBar } from "@vibe/core";
     {
       positive: {
         component: (
-          <DialogContentContainer className="monday-storybook-breadcrumbs-bar_inline-wrapper">
+          <DialogContentContainer
+            style={{
+              display: "flex",
+              alignItems: "center",
+              padding: "var(--space-8) var(--space-12)",
+              gap: 12,
+              width: 320
+            }}
+          >
             <Avatar size="medium" src={person3} type="img" />
-            <div className="monday-storybook-breadcrumbs-bar_column-wrapper">
-              <span className="monday-storybook-breadcrumbs-bar_title">Rotem Dekel</span>
+            <div className={styles.list}>
```

Comment:
Why is it needed?

Reply:
check the old file
it had style for `ol`

I've wrote in the PR description:
> I had no choice but to use `className` for the `ol` styling. The story itself is kind of a "foul"